docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -177,13 +188,13 @@ The prebuilt workspaces feature has these current limitations:
 
   Prebuilt workspaces can only be used with the default organization.
 
-  [coder/internal#364](https://github.com/coder/internal/issues/364)
+  [View issue](https://github.com/coder/internal/issues/364)
```

Comment:
I think it looks nicer in the GitHub link format - it's descriptive and familiar

like this - I'll paste this link below as `https://github.com/coder/internal/issues/364` and GitHub does the reformatting thing:

https://github.com/coder/internal/issues/364

Reply:
There was some hesitation in the bug bash call around showing `coder/internal` links because folks might think they're not accessible.

Reply:
oh interesting

works for me either way